<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font: italic 11pt &quot;Comic Sans MS&quot;;
background-color: rgb(255, 255, 255);

}
QSlider::handle:horizontal{background-color:rgb(123, 82, 167);}



QPushButton{
	background-color:#ededed;
	border-radius:15px;
	border:1px solid #d6bcd6;
	padding:7px 25px;

}
QPushButton#Identify{border-radius:50px}
QPushButton:hover {
	
	
	
	background-color: rgb(217, 217, 217);
}
QPushButton:pressed{
	position:absolute;
	top:-5px;
	border-style: ridge;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="Browse2">
        <property name="maximumSize">
         <size>
          <width>230</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse Song 2</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/music.png</normaloff>Icons/music.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="firstSongName">
        <property name="styleSheet">
         <string notr="true">
color: rgb(123, 82, 167);
color: rgb(108, 73, 148);</string>
        </property>
        <property name="text">
         <string>Song Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="Browse1">
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse Song 1</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/music.png</normaloff>Icons/music.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="secondSongName">
        <property name="styleSheet">
         <string notr="true">color: rgb(123, 82, 167);
color: rgb(108, 73, 148);</string>
        </property>
        <property name="text">
         <string>Song Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QSlider" name="mixerSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="mixerValue">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(123, 82, 167);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QPushButton" name="Identify">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/search (1).png</normaloff>Icons/search (1).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="resultsTable"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mixerSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mixerValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
